src/Cli/Microsoft.DotNet.Cli.Utils/Command.cs

```diff
@@ -78,7 +78,7 @@ public CommandResult Execute(Action<Process>? processStarted)
                     LocalizableStrings.ProcessExitedWithCode,
                     FormatProcessInfo(_process.StartInfo),
                     exitCode,
-                    sw.ElapsedMilliseconds);
+                    sw?.ElapsedMilliseconds);
```

Comment:
nit: we know this is non-null in this case:
```suggestion
                    sw!.ElapsedMilliseconds);
```

Reply:
FWIW, we got strong guidance to avoid ! where possible by adding null checks. That being said, this seems like the compiler should know that this isn't null here and adding a null check seems incorrect as well. @JeremyKuhne in case he has advise or if this is the exception to that no ! guidance.

Reply:
In this case you can add a `Debug.Assert(sw is not null);` ahead of it. That will handle the case where someone changes the logic to where this is no longer true. @jaredpar who is the right person to ping about null analysis?

Reply:
It appears this [project multi-targets](https://github.com/dotnet/sdk/blob/main/src/Cli/Microsoft.DotNet.Cli.Utils/Microsoft.DotNet.Cli.Utils.csproj#L9)between `net472` and other TFMS but runs nullable analysis on all of them. That is the setup I advised against when I went by your desk @marcpopMSFT because it leads to situations like this. `Debug.Assert` has _no impact_ on nullable analysis when targeting `net472` because it doesn't have the new attributes. 

Strongly recommend disabling nullable warnings on .NET Framework when you mulit-target with other TFMS.

Reply:
@jaredpar I tried adding your suggestion of disabling nullable warnings for MT netfx builds but that doesn't work: https://github.com/dotnet/sdk/pull/45420

I can confirm locally that DisableNullableWarnings is true in the netfix CLI.Utils build but I still get this warning: `C:\repos\sdk\src\Cli\Microsoft.DotNet.Cli.Utils\Command.cs(82,21): error CS8602: Dereference of a possibly null reference. [C:\repos\sdk\src\Cli\Microsoft.DotNet.Cli.Utils\Microsoft.DotNet.Cli.Utils.csproj]`